app/styles/modules/_branding.scss

```diff
@@ -90,6 +90,10 @@
     padding-bottom: 10px;
   }
 
+  #submit-btn {
+      padding-top: 10px;
```

Comment:
@himanish-star I think what you need here is `margin-top: 10px`. Using `padding-top` will cause the button text to be pushed down. Also, our css use a different style than code, mind removing extra space in front and have it line up with other lines?

Reply:
@vbudhram, on it.

Reply:
I have made the changes